### ISSUE TITLE
Handle app refresh gracefully

### ISSUE THREAD

**Issue by loopthing** [loopthing/bearded-legends#3]
For now, we swipe down on About page to refresh the app, but we should give a button to do this for accessibility. Also, while reloading, the page briefly flickers. We should handle this better as well and coordinate with dark/light mode preferences.

The lack of button is an accessibility issue, so I'm labeling this "bug".

**Comment by loopthing** [comment]
A little better in v1.0.2 but not perfect. We still need to give visual feedback while updating so that the user knows the attempt to update did something. 

The rest of the work is enhancements, so I'm dropping the bug label.

**Comment by loopthing** [comment]
I left some comments about visual feedback on a different ticket. Using this one for update functionality.

The update functionality is not implemented. The app is very aggressive about getting the latest version right now because I am trying to overcome browser caching, which is not playing nicely with the caching policy originally implemented in service workers. Need to reconsider the whole thing.
